### PR TITLE
feat: add NewRelic Integration (#333)

### LINKED ISSUE #333: ICM: Add possibility to integrate NewRelic Agent
Make APM available for developers.

### PR DESCRIPTION
**Pull request by Thomas-Bergmann**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] CI-related changes
- [ ] Documentation content changes
- [ ] Application / infrastructure changes

## Release ##

Be sure to include PR label `major`, `minor` or `patch` when merging into `main`. This determines which part of the semantic version number needs to be bumped automatically.

## What Is the Current Behavior?

With ICM-AS 11.3.0 container provides New Relic integration of java agent. This new feature enable the activation of that feature via helm charts. Older versions of ICM-AS can use the new helm chart with deactivated feature.

Issue Number: Closes #333 

## Does this PR Introduce a Breaking Change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: charts/icm-as/tests/newrelic_test.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      name: icm-as
+    chart:
+      version: 0.8.15
+    values:
+      - ../values.yaml
+    set:
+      newrelic.enabled: true
+      newrelic.license_key: "secret-license-test"
+    template: config-newrelic-yaml-cm.yaml
+#data:
```
Comment (cortlepp-intershop) [inline charts/icm-as/tests/newrelic_test.yaml:34]: Why is this commented out? It seems like an important part of the test which should be checked.

File: charts/icm-as/values.yaml [hunk 2]
```diff
@@ -72,6 +72,14 @@ datadog:
   # define (additional) datadog options
   additionalOptions:
 
+# configuration related to New Relic
+newrelic:
```
Comment (cortlepp-intershop) [inline charts/icm-as/values.yaml:76]: Maybe consider also adding this section to the `values.yaml` in the `icm` umbrella chart.

File: charts/icm-as/tests/newrelic_test.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Thomas-Bergmann) [reply charts/icm-as/tests/newrelic_test.yaml:34]: if you have a better idea to test a config map. please enhance the test.

[the remark this answers is not in the document]
Comment (cortlepp-intershop) [reply charts/icm-as/tests/newrelic_test.yaml]: I added a test which checks for the presence of the license key in the data.

File: charts/icm-as/README.md [hunk 3]
```diff
@@ -81,8 +81,10 @@ Prerequisites are:
 
 * [helm-unittest](https://github.com/helm-unittest/helm-unittest)
 
+Please check the unit tests before pushing changes.
+
 ```bash
-helm unittest --helm3  --output-file unit.xml --output-type JUnit charts/icm-as
```
Comment (skiesewetter-intershop) [inline charts/icm-as/README.md:85]: Why did you change the output target and type?

**Review by skiesewetter-intershop — CHANGES_REQUESTED** [review]

**Review by khauser — APPROVED** [review]

**Review by khauser — DISMISSED** [review]

**Pull request merged**